nearcore/src/lib.rs

```diff
@@ -124,6 +118,12 @@ fn apply_store_migrations(path: &Path, near_config: &NearConfig) -> anyhow::Resu
         "DB version {db_version} is created by a newer version of neard, \
          please update neard"
     );
+    anyhow::ensure!(
+        db_version >= 27,
+        "DB version {db_version} is created by an ancient version of neard and \
+         is no longer supported by this version, please migrate using older \
+         releases"
```

Comment:
Do we have some sort of a table/document/documentation which would list the correspondence between database version and the neard release required to upgrade it? Could it be linked from here? Or alternatively could this message say which versions of neard to run to upgrade in the first place?